src/ess/livedata/dashboard/job_orchestrator.py

```diff
@@ -110,27 +109,20 @@ def __init__(
         """
         self._command_service = command_service
         self._workflow_config_service = workflow_config_service
-        self._source_names = source_names
         self._workflow_registry = workflow_registry
         self._config_store = config_store
         self._logger = logging.getLogger(__name__)
 
         # Workflow state tracking
         self._workflows: dict[WorkflowId, WorkflowState] = {}
 
+        # Workflow subscription tracking
+        self._subscriptions: dict[SubscriptionId, Callable[[JobNumber], None]] = {}
```

Comment:
Can you explain the difference between a subscription id and a workflow id?

Reply:
Workflow ID identifies a workflow. Subscription ID will be used, e.g., to unsubscribe. Since we may have multiple plots for a workflow we cannot repurpose the workflow ID for this.